impeller/renderer/backend/vulkan/swapchain_impl_vk.cc

```diff
@@ -10,10 +10,12 @@
 #include "impeller/renderer/backend/vulkan/formats_vk.h"
 #include "impeller/renderer/backend/vulkan/surface_vk.h"
 #include "impeller/renderer/backend/vulkan/swapchain_image_vk.h"
+#include "vulkan/vulkan_structs.hpp"
 
 namespace impeller {
 
 static constexpr size_t kMaxFramesInFlight = 3u;
+static constexpr size_t kPollFramesForOrientation = 1u;
```

Comment:
nit: document what this means (i.e. 1 is every frame, 2 is every other frame)

Reply:
Done!